pages/account/statistics.js

```diff
@@ -113,6 +123,17 @@ export default function Statistics({ data, profile, progress }) {
     };
   });
 
+  const profileSectionsSpecification = {
+    "bio": { value: "bio", link: "https://linkfree.eddiehub.io/docs/how-to-guides/bio", description: "Describe in a few words who you are.", component: Bio },
+    "links": { value: "links", link: "https://linkfree.eddiehub.io/docs/how-to-guides/links", description: "Let people discover all your great content in one place.", component: Links },
+    "milestones": { value: "milestones", link: "https://linkfree.eddiehub.io/docs/how-to-guides/milestones", description: "Demonstrate the highlights of your career.", component: Milestones },
+    "tags": { value: "tags", link: "https://linkfree.eddiehub.io/docs/how-to-guides/tags", description: "These are searchable keywords for people to discover your Profile.", component: Tags },
+    "socials": { value: "socials", link: "https://linkfree.eddiehub.io/docs/how-to-guides/socials-shortcuts", description: "Add a shortcut to your favourite social media accounts.", component: Socials },
+    "location": { value: "location", link: "https://linkfree.eddiehub.io/docs/how-to-guides/events", description: "Add events you either hosted or attended.", component: Location },
+    "testimonials": { value: "testimonials", link: "https://linkfree.eddiehub.io/docs/how-to-guides/testimonials", description: "Show off the great feedback you have received.", component: Testimonials },
+  }
```

Comment:
Please check you have VScode Prettier installed, so the formatting automatically matches the projects